batchflow/tests/regression_metrics_test.py

```diff
@@ -0,0 +1,85 @@
+""" Tests for regression metrics """
+# pylint: disable=import-error, no-name-in-module
+import pytest
+import numpy as np
+
+from batchflow.models.metrics import RegressionMetrics
+
+PARAMS_DEFINED = [(None, 'mae', [1.5, 0.25]),
+                  (None, 'mse', [3, 0.25]),
+                  (None, 'median_absolute_error', [1, 0.5]),
+                  (None, 'max_error', [2, 1]),
+                  (None, 'rmse', [1.73, 0.5]),
+                  (None, 'r2', [-15, -0.33]),
+                  (None, 'explained_variance_ratio', [-3, 0]),
+                  (None, 'acc', [1, 1]),
+                  ('mean', 'mae', 0.875),
+                  ('mean', 'mse', 1.625),
+                  ('mean', 'median_absolute_error', 0.75),
+                  ('mean', 'max_error', 1.5),
+                  ('mean', 'rmse', 1.12),
+                  ('mean', 'r2', -7.67),
+                  ('mean', 'explained_variance_ratio', -1.5),
+                  ('mean', 'acc', 1)]
+
+@pytest.mark.parametrize('targets, predictions, weights', [([[1, 1], [2, 2]], [[1, 0], [4, 2]], [1, 3])])
+@pytest.mark.parametrize('output_agg, metric_name, exp_res', PARAMS_DEFINED)
+def test_defined_values(targets, predictions, weights, output_agg, metric_name, exp_res):
+    """ Test that expected values match the real ones for some non border values """
```

Comment:
In file named `regression_metrics_test` that is located in directory `tests` you really don't want to use the word `test` in a docstring, as well as saying that it compares expected values and calculated ones